sys/riscv/riscv/locore.S

```diff
@@ -303,29 +529,35 @@ ENTRY(freebsd64_sigcode)
 	ecall
 
 	j	1b
-END(freebsd64_sigcode)
 	/* This may be copied to the stack, keep it 16-byte aligned */
 	.align	3
+END(freebsd64_sigcode)
 freebsd64_esigcode:
 .option pop
 #endif
 
 	.data
 	.align	3
 	.global	szsigcode
+	.type szsigcode, @object
 szsigcode:
 	.quad	esigcode - sigcode
+	.size szsigcode, . - szsigcode
```

Comment:
```suggestion
	.size	szsigcode, . - szsigcode
```

Reply:
I did a general sweep to fix these after the first one above for pagetable_l2 or the like.